Makefile

```diff
@@ -199,3 +199,28 @@ go-lint: go-lint-install ## linter config in file at root of project -> '.golang
 fix: go-lint-install ## try to fix automatically issues
 	go mod tidy
 	golangci-lint run --fix
+
+.PHONE: app
+app: ## Run the app
+	go run main.go
+
+.PHONY: ui
+ui: ## Run the ui
+	cd ui && npm install && npm start
+
+.PHONE: webapp
+webapp: ## Run App and UI
+	 @$(MAKE) -j ui app
+
+.PHONY: clean-ports
```

Comment:
Not required if random available ports are assigned in the app and stored in environment variables.